docs/api.md

```diff
@@ -8,11 +8,11 @@ title: Kubernetes API Overview
 
 Primary system and API concepts are documented in the [User guide](/docs/user-guide/).
 
-Overall API conventions are described in the [API conventions doc](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/api-conventions.md).
+Overall API conventions are described in the [API conventions doc](https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md).
 
```

Comment:
Please revert this kind of change to use the relative path by assignment expression in github-markdown.